fitbenchmarking/controllers/ceres_controller.py

```diff
@@ -121,6 +121,12 @@ def setup(self):
 
         self.ceres_options.max_num_iterations = 10000
 
+        if self.value_ranges is not None:
+            value_ranges_lb, value_ranges_ub = zip(*self.value_ranges)
+            for i, (l, u) in enumerate(zip(value_ranges_lb, value_ranges_ub)):
```

Comment:
I think the 2 zips cancel each other out?
```
a = [[1, 'a'],
     [2, 'b'],
     [3, 'c']]
ls, rs = zip(*a) # == zip([1,'a'], [2, 'b'], [3, 'c'])
                 # == [1, 2, 3], ['a', 'b', 'c']
b = zip(ls, rs) # == zip([1, 2, 3], ['a', 'b', 'c'])
                # == [[1, 'a'], [2, 'b'], [3, 'c']]
b == a
```
So you should just be able to enumerate over `self.value_ranges`?